content/20.treatments.md

```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
+One type of stimulus which research indicates can induce trained immunity is BCG vaccination (BCG is an acronym for bacillus Calmette-Guerin).
+BCG is a live attenuated strain derived from an isolate of Mycobacterium bovis with the common reason for application being prevention of tuberculosis in humans.
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination.
+Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
+One set of trials for example has time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
+An article [@doi:10.1016/j.cell.2020.04.042] that very briefly summarizes the setup of these trials also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many other possible factors or confounding factors (country age distribution, detection efficiency, stochastic epidemic dynamic effects, differences in healthcare capacity over time in relation to epidemic dynamics, many other mundane factors) and (2) also it is unclear what is the relation between BCG vaccination at different ages and at longer times before the start of the SARS-CoV-2 epidemic and BCG vaccination at different ages during the SARS-CoV-2 epidemic at shorter times before the risk of possible infection; at least some of these considerations are also made or pointed out in the data analyses sited and in the article.
+The article [@doi:10.1016/j.cell.2020.04.042] also includes various related considerations (generally related animal and human studies, general safety, specific possible safety considerations with respect to SARS-CoV-2).
+Additionally, we review here one of the most important considerations: severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009; @doi:10.1016/j.cell.2020.04.042; @doi:10.1016/j.chom.2020.05.008]); also, trained immunity itself may sometimes be related to either balanced, or too much or too little, or useful or harmful immunological responses.
```

Comment:
Note: I did not look up these papers but this seems reasonable!

Reply:
For breaking up the sentence, we can just remove the semi-colon & make it 2 sentences instead of 1 -- i think that's simpler than verifying whether our two versions of the 2nd clause are the biologically same. I like the sentence you added afterwards -- it puts additional useful points to what I wrote.
*Reminder* to partly incorporate feedback.